packages/ColorPicker/src/__snapshots__/ColorPicker.test.tsx.snap

```diff
@@ -9,11 +9,13 @@ exports[`ColorPicker It should render a snapshot 1`] = `
     <button
       class="ids-color-picker__button"
       data-color="dandelion200"
+      data-rac=""
```

Comment:
What is it used for?

Reply:
This is a react-aria button so I'm guessing it's added by them. I double checked the code, not something we added.

Reply:
react-aria-components (rac) seems to be adding this property on react aria components. 

There is a major release releasing todayt of react-aria-components, and the code that adds the data-rac is still there. I think it's used by their tailwindcss integration to target react-aria-components. 

https://github.com/adobe/react-spectrum/blob/9ce2f674eab2cc8912800d3162dcf00a1ce94274/packages/react-aria-components/src/utils.tsx#L134-L135

Reply:
in all cases, don't worry about it for now i think